administrator/components/com_content/views/article/tmpl/edit.php

```diff
@@ -181,11 +181,15 @@
 	</div>
 </form>
 <?php
+JHtml::_('jquery.framework', false);
 $document = JFactory::getDocument();
 $document->addScriptDeclaration('
-function copytext() {
-textBox = document.getElementById("articletitle");
-message = document.getElementById("menutitle");
-message.value = textBox.value;
+$(document).ready(function () {
+$("input[name=title]").keyup(function () {
+var a = $("input[name=title]").val();
+var b = a;
+$("input[name=menutitle]").val(b);
+});
```

Comment:
@shubhamnba2009 $document->addScriptDeclaration, adds the code to the <head> section of the HTML. But you are more severe issue, then just indenting. 

You can't just use $ here.. There is still Mootools (or possible other js frameworks) in Joomla!.

So either use jQuery instead of $, like `jQuery(document)`...

or:

```
(function ($) {
   $(document).ready(...
}(jQuery));
```